spec/models/rbac_spec.rb

```diff
@@ -1,7 +1,10 @@
 describe Rbac do
   before { allow(User).to receive_messages(:server_timezone => "UTC") }
+
   let(:default_tenant) { Tenant.seed }
 
+  let(:admin_user)   { FactoryGirl.create(:user, :role => "super_administrator") }
+
```

Comment:
Is there a reason these `lets` have inconsistent whitespace?  I can see separating the `before` from the `lets` but not this one unless I'm missing something.  Also, verify align the `{` for all of the `lets` if you group them together.